PythonToolbox/quantconnect/LeanReportCreator.py

```diff
@@ -0,0 +1,423 @@
+                #project-metadata tr > td:last-child {
+                    text-align: center;
+                }
+    
+                #key-statistics tr > td:last-child {
+                    text-align: right;
+                }
+    
+                #key-statistics tr > td,
+                #key-characteristics tr > td,
+                table.table tr > td {
+                    vertical-align: middle;
+                }
+    
+                table.table.align-top tr > td {
+                    vertical-align: top;
+                }
+    
+                .table.qc-table.compact {
+                    height: 234px;
+                }
+    
+                .table.qc-table.compact img {
+                    width: 100%;
+                    height: 100%;
+                    max-height: 196px;
+                }
+    
+                #author-box img {
+                    max-width: 33%;
+                    height: auto;
+                    max-height: 33%;
+                    float: left;
+                    margin-right: 5px;
+                    margin-bottom: 5px;
+                }
+    
+                .header .header-title {
+                    margin-top: 40px;
+                    line-height: 30px;
+                    position: absolute;
+                    left: 0;
+                    right: 0;
+                    text-align: center;
+                }
+    
+                #strategy-description,
+                #author-bio {
+                    max-height: 194px;
+                    word-wrap: break-word;
+                    height: 100%;
+                }
+
+            </style>
+        </head>
+        <body>''' + downloadButton + '''
+        <div class="page">
+            <div class="header">
+                <div class="header-left">
+                    <img src="https://cdn.quantconnect.com/web/i/logo.png">
+                </div>
+                <div class="header-title">Strategy Report Summary</div>
+                <div class="header-right">''' + self.user['projectName'] + '''</div>
+            </div>
+            <div class="content">
+                <h1 class="hidden">Strategy Report</h1>
+                <div class="container-row">
+                    <div class="col-xs-8">
+                        <table id="description-box" class="table qc-table compact no-margin align-top">
+                            <thead>
+                            <tr>
+                                <th>
+                                    Strategy Description
+                                </th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            <tr>
+                                <td>
+                                    <p id="strategy-description" class="text-justify editable" style="max-width: 590px;overflow: hidden;">''' + self.user['projectDescription'] + '''</p>
```

Comment:
If the box is empty, it looks shorther than the bio to fix replace `style="max-width: 590px;overflow: hidden;"` with `style="max-width: 590px;overflow: hidden;height:194px;"`